app/src/renderer/js/editor.js

```diff
@@ -105,6 +107,13 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  function getTrimmedVideoDuration() {
+    const inValue = getTrimmerValue(trimmerIn);
+    const outValue = getTrimmerValue(trimmerOut);
+    currentPreviewDuration = outValue - inValue;
```

Comment:
this function mutates the external var `currentPreviewDuration`, better to make this one just 
```
return outValue - inValue
```
and mutate it where you call it further down
```
currentPreviewDuration = getTrimmedVideoDuration()
```

Reply:
@albinekb Can you do a follow-up PR with some improvements maybe?

Reply:
Sorry, I missed this entirely.